tests/lib/im_helper.rb

```diff
@@ -70,17 +78,43 @@ def download_and_uncompress(host, urllink, compressed_file, uncompress_to)
 def verify_im_installed?(installed_directory)
   step "Verify IBM Installation Manager is installed into directory: #{installed_directory}"
   step 'Verify 1/3: IBM Installation Manager Launcher'
-  if agent.file_exist?("#{installed_directory}/eclipse/launcher") == nil
+  if agent.file_exist?("#{installed_directory}/eclipse/launcher") == false
     fail_test "Launcher has not been found in: #{installed_directory}/eclipse"
   end
 
   step 'Verify 2/3: IBM Installation Manager License File'
-  if agent.file_exist?("#{installed_directory}/license/es/license.txt") == nil
+  if agent.file_exist?("#{installed_directory}/license/es/license.txt") == false
     fail_test "License file has not been found in: #{installed_directory}/license"
   end
 
   step 'Verify 3/3: IBM Installation Manager Version'
-  if agent.file_exist?("#{installed_directory}/properties/version/IBM_Installation_Manager.*") == nil
+  if agent.file_exist?("#{installed_directory}/properties/version/IBM_Installation_Manager.*") == false
     fail_test "Version has not been found in: #{installed_directory}/properties/version"
   end
 end
+
+# Get the source links for the IBM Installation manager installer
+# either for Linux or AIX agent
+#
+# ==== Attributes
+#
+# ==== Returns
+#
+# +The links to IBM Installation Manager installer or nil+
+#
+# ==== Raises
+#
+# +none+
+#
+# ==== Examples
+#
+# get_resource_link
+def get_source_link(host)
+  source = nil
+  if (host['platform'] =~ /aix/)
+    source = 'http://int-resources.ops.puppetlabs.net/QA_resources/ibm_websphere/agent.installer.aix.gtk.ppc_1.8.4000.20151125_0201.zip'
+  elsif (host['platform'] =~ /linux/)
+    source = 'http://int-resources.ops.puppetlabs.net/QA_resources/ibm_websphere/agent.installer.linux.gtk.x86_64_1.8.3000.20150606_0047.zip'
+  end
+  return source
```

Comment:
You can save yourself some lines of code without reducing readability by doing this:

```
if (host['platform'] =~ /aix/)
+    return 'http://int-resources.ops.puppetlabs.net/QA_resources/ibm_websphere/agent.installer.aix.gtk.ppc_1.8.4000.20151125_0201.zip'
+  elsif (host['platform'] =~ /linux/)
+    return 'http://int-resources.ops.puppetlabs.net/QA_resources/ibm_websphere/agent.installer.linux.gtk.x86_64_1.8.3000.20150606_0047.zip'
+  end
```